packages/contracts/deployments/engine/V3/studio/dev/DEPLOYMENTS.md

```diff
@@ -0,0 +1,91 @@
+# Engine Factory Deployment
+
+Date: 2024-05-08T17:14:51.615Z
+
+## **Network:** sepolia
+
+## **Environment:** dev
+
+**Engine Implementation:** https://sepolia.etherscan.io/address/0x00000000BB846ED9fb50fF001C6cD03012fC4485#code
+
+**Engine Flex Implementation:** https://sepolia.etherscan.io/address/0x00000000B33F6D5cA8222c87EAc99D206A99E17E#code
+
+**Engine Factory:** https://sepolia.etherscan.io/address/0x0000000f84351b503eB3Df72C7E1f169b2D32728#code
+
+**Core Registry:** https://sepolia.etherscan.io/address/0x985C11541ff1fe763822Dc8f71B581C688B979EE#code
+
+Ownership on Core Registry transferred to the Engine Factory
+
+---
+
+# Batch Engine and Engine Flex Contract Deployments
+
+Date: 2024-05-09T17:01:48.023Z
+
+## **Network:** sepolia
+
+## **Environment:** dev
+
+**Engine Implementation:** https://sepolia.etherscan.io/address/0x00000000BB846ED9fb50fF001C6cD03012fC4485#code
+
+**Engine Flex Implementation:** https://sepolia.etherscan.io/address/0x00000000B33F6D5cA8222c87EAc99D206A99E17E#code
+
+**Engine Factory:** https://sepolia.etherscan.io/address/0x0000B005007298838aCF6589d4342920A9cB002a#code
+
+**Core Registry:** https://sepolia.etherscan.io/address/[object Promise]#code
+
+---
+
+**AdminACL:** https://sepolia.etherscan.io/address/0x37935f04d040F564337AE9BD76f2668664Fc9604#code
+
+        **Engine Contract Type:** Engine
```

Comment:
since this will be one big file, maybe this heading could just be something a bit more look-upable?
wdyt about something like:
```suggestion
## Deployment: Engine | dev-test2 (DEV_TEST2) | 0xce9Fe...Ae85De
```

Reply:
Good call! I updated the template to:
`### Deployment: <Engine Contract Type> | <Contract Address>`